data/categories.txt

```diff
@@ -4538,6 +4538,123 @@ sk:Kasíno
 sw:Kasino|burudani
 fa:قمارخانه|کازینو
 
+amenity-gambling
+en:Gambling
```

Comment:
Probably we should add separate category ```@gambling``` and assign it on casino, bookmaker, gambling, adult_gaming

Reply:
Want to see all the "trash" with one query :)

Reply:
Not in the app's UI, please. Let it remain in search-queries only.

Reply:
Of course

Reply:
Added category to search